.scalafmt.conf

```diff
@@ -1,4 +1,7 @@
 version = "2.7.5"
 preset = Scala.js
-maxColumn = 120
-trailingCommas = always
+maxColumn = 100
```

Comment:
Why this reduction from 120 to 100? I like big columns :stuck_out_tongue_closed_eyes:

Reply:
You are in good company with Igor then! I thought I could the Scala Style Guide as a pretext for inching things towards my preference for narrower code :stuck_out_tongue_winking_eye: 

If you and/or others on the team would prefer it stay at 120, I'll revert this bit :)

Reply:
I've put this back to 120 :)